### PR TITLE
UCX: bump version to v1.3.1

### PR DESCRIPTION
**Pull request by amaslenn**

### CODE REVIEW

**Comment by amaslenn** [comment]
@yosefe could you please fill NEWS for this version?

File: doc/doxygen/header.tex [hunk 1]
```diff
@@ -128,7 +128,7 @@
     {\Large Unified Communication X (UCX)}\\
 \vspace*{0.5cm}
 {\large API Standard}\\
-{\small Version 1.3}\\
+{\small Version 1.3.1}\\
```
Comment (yosefe) [inline doc/doxygen/header.tex]: API does not contain minor version, since API does not change between minor versions

File: NEWS [hunk 2]
```diff
@@ -7,6 +7,11 @@
 ##
 #
 
+## 1.3.1 (August 20, 2018)
+
+TBD
```
Comment (yosefe) [inline NEWS]:
```
Bugfixes:
- Prevent potential out-of-order sending in shared memory active messages
- CUDA: Include cudamem.h in source tarball, pass cudaFree memory size
- Registration cache: fix large range lookup, handle shmat(REMAP)/mmap(FIXED)
- Limit IB CQE size for specific ARM boards
- RPM: explicitly set gcc-c++ as requirement
```

**Comment by shamisp** [comment]
Is this a drop for redhat ?

**Comment by yosefe** [comment]
yes, fedora updated the spec file requirements

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7484/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7488/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@shamisp pls take a look

**Comment by shamisp** [comment]
👍 
Let's tag it and send announcement on the list

**Pull request merged**